volume/local/local.go

```diff
@@ -144,22 +152,27 @@ type Volume struct {
 	driverName string
 }
 
+// Name returns the volume name
 func (v *Volume) Name() string {
 	return v.name
 }
 
+// DriverName returns the driver that created the volume.
 func (v *Volume) DriverName() string {
 	return v.driverName
 }
 
+// Path returns the data location.
 func (v *Volume) Path() string {
 	return v.path
 }
 
+// Mount returns the location where data is.
```

Comment:
Mount returns the data location and is not really used for this driver, and is only here to satisfy the `Volume` interface.